src/pages/token/rewards/RewardsPage.tsx

```diff
@@ -66,6 +64,7 @@ const RewardsPage = () => {
 
   const stakingRewardPanel = (
     <StakingRewardPanel
+      validators={validators?.toArray() ?? []}
```

Comment:
nit:
I wonder if it would be better to get the `validators` directly in `src/views/dialogs/StakingRewardDialog.tsx`, instead of piping it down as a prop.

Since `StakingRewardDialog` is already connected to redux with `useAppSelector` anyways.

Reply:
ah yess makes sense to me - will update